src/components/card/card.js

```diff
@@ -0,0 +1,45 @@
+import './card.css'
+import woman from '../../images/illustration-woman-online-desktop.svg';
+import Question from '../question/question';
+
+
+const Card = ()=>{
+    return(
+        <div className="card-wrapper">
+            <div className="card">
+                <div className="left-side">                  
+                    <img className="woman" src={woman} alt="woman"/>
+                </div>
+                <div className="right-side">
+                    <b className="FAQ">FAQ</b>
+                    <div className="questions">
+                        <Question  
```

Comment:
https://github.com/meltaway/dsc-studygroup-react/blob/95229d4cf5e28b02507ca446665e71b1250837b3/src/components/Accordion.js
Check out this way of avoiding duplication